packages/react-widgets/src/widgets/TableWidget.js

```diff
@@ -34,7 +34,6 @@ function TableWidget({
   onError,
   initialPageSize = 10,
   onPageSizeChange,
-  global,
```

Comment:
I guess removing global option is a product decision, already discussed, right?. From Builder it's not used, so ok from that side, but not sure if we might have other uses cases outside... @aaranadev ?

Reply:
Global mode is not supported for table widgets, according to both documentation and actual way it's integrated in Builder, where there's no global radio button.

https://docs.carto.com/carto-for-developers/carto-for-react/library-reference/widgets#tablewidget

Reply:
If instead we want to have it, it should be documented, and then I'll revert and add documentation.

Reply:
I'll revert this to be on the safe side.